include/swift/Runtime/Metadata.h

```diff
@@ -1037,6 +1037,12 @@ SWIFT_RUNTIME_STDLIB_SPI
 void _swift_registerConcurrencyStandardTypeDescriptors(
     const ConcurrencyStandardTypeDescriptors *descriptors);
 
+/// Walk the parent descriptor chain to find the nominal type parent of the
+/// provided type.
```

Comment:
Can you document the limitation of not handling generic types here? That might save someone some confusion.